seonshine_frontend/src/pages/signUp/ChooseUserType/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
```

Comment:
move `ChooseUserType, PendingApproval, ProfileRegistration` to `components` folder inside `signUp` folder pages